chemprop/nn/predictors.py

```diff
@@ -294,18 +297,16 @@ class MulticlassDirichletFFN(MulticlassClassificationFFN):
     _T_default_metric = CrossEntropyMetric
 
     def forward(self, Z: Tensor) -> Tensor:
-        Y = super().forward(Z).reshape(len(Z), -1, self.n_classes)
+        Y = super().train_step(Z)
 
-        Y = Y.softmax(-1)
-        Y = F.softplus(Y) + 1
+        alpha = F.softplus(Y) + 1
 
```

Comment:
The uncertainty can be calculated as `u = self.n_classes / alpha.sum(-1)`. However, since the contract we signed specifies 1 target, I only return the predicted property. Additionally, as we discussed here (https://github.com/chemprop/chemprop/pull/959#issuecomment-2261639212), the multiclass classification output should be in the shape of b x t x c.